services/datamanager/builtin/builtin.go

```diff
@@ -532,6 +556,15 @@ func (svc *builtIn) Reconfigure(
 			svc.closeSyncer()
 		}
 	}
+
+	if !svc.captureDisabled {
```

Comment:
Can we just add a comment here explaining this behavior? If capture is enabled, we want to kick off a go routine to check if disk space is fulling up due to viam?

Reply:
Done!